src/ast/ast.h

```diff
@@ -199,6 +199,12 @@ class Builtin : public Expression {
   std::string ident;
   int probe_id;
 
+  bool is_argx() const
+  {
+    return !ident.compare(0, 3, "arg") && ident.size() == 4 &&
```

Comment:
hmm, this could use a comment as to what all these numbers are - maybe just an example - I know you didn't change this logic but would be good to add.

Reply:
Good idea, added.